File: backend/models/sub_feature.go
package models

import (
	"time"
)

type SubFeature struct {
	ID          int       `json:"id" gorm:"primaryKey"`
	FeatureID   int       `json:"feature_id"`
	Title       string    `json:"title"`
	Description string    `json:"description"`
	Status      string    `json:"status"`
	Priority    string    `json:"priority"`
	AssigneeID  int       `json:"assignee_id"`
	CreatedAt   time.Time `json:"created_at"`
	UpdatedAt   time.Time `json:"updated_at"`
}

File: frontend2/src/app/types/subfeature.ts
export interface SubFeature {
  id: number;
  feature_id: number;
  title: string;
  description: string;
  status: 'todo' | 'in_progress' | 'done';
  priority: 'low' | 'medium' | 'high';
  assignee_id: number;
  created_at: string;
  updated_at: string;
} 